apps/metricstream/template.yaml

```diff
@@ -224,6 +226,7 @@ Resources:
                 Resource: !GetAtt 'DeliveryStream.Arn'
   MetricStream:
     Type: AWS::CloudWatch::MetricStream
+    Condition: NotDeployLambda
```

Comment:
do we need to add a whole variable for this? can't we just do `!DeployLambda`

Reply:
I tried `Condition !Not DeployLambda` and the sam app threw an error - it seems it wants the condition to be a string that refers to an existing condition.